docs/changelog/0.4.11.md

```diff
@@ -0,0 +1,178 @@
+
+# 0.4.11 (2023-03-15)
+
+We're happy to announce the release of Scala Native. It's the next maintenance release for Scala Native 0.4.x. As always it brings bug fixes and minor improvements.
+
+## Notable changes
+
+### Extern methods with a variadic number of arguments
+For a long time, Scala Native supported C `va_list` using `scalanative.unsafe.CVarArgList`. This allowed for interop with some of the C functions taking the variadic number of arguments. This release makes usage and definition of them easier, by restoring support for idiomatic ways of passing them using Scala variadic arguments lists.
+
+### Support for LLVM 15
+The latest versions of LLVM added a new internal representation of pointers - their opaque variant replaces typed pointers. This change should not affect the used of the LLVM toolchain, but in some specific builds it could have lead to linking issues.
+Now Scala Native will try to detect version of used LLVM toolchain. When using LLVM 15 or newer Scala Native toolchain would always generate opaque pointers in the compiled LLVM IR. 
```

Comment:
s/version of used/version of LLVM toolchain/

or "version of LLVM toolchain used by the operating system shell"

or better bikeshedding.